cfpb/complaint-intake
# 72 affiliations

## Changes

- Moves affiliations section to primary consumer section.
- Makes affiliations checkboxes show current status sections concurrently.

## Testing

- `gulp clean && gulp build`
- Visit /your-information.html
- Check `The primary consumer doesn't have an email address`.
- Check `A servicemember or veteran` and `The spouse or dependent of a servicemember or veteran` checkboxes.

## Review

- @niqjohnson 

## Screenshots

![screen shot 2016-07-06 at 1 03 16 pm](https://cloud.githubusercontent.com/assets/704760/16627263/6ed1a06a-437a-11e6-9e33-cb95f4677312.png)


## Notes

- Contains https://github.com/cfpb/complaint-intake/pull/41
- Only updates primary consumer. Additional consumer and additional point of contact will be updated in a separate PR.
